regolith/builders/grantreportbuilder.py

```diff
@@ -70,6 +77,12 @@ def latex(self):
 
         # Plans for Next Reporting Period to Accomplish Goals
 
+        # Individuals that have worked on project
+        individuals_data = []
+        for p in people:
```

Comment:
you keep looping over people so why not put each of these actions inside a single outer loop over people?